Incompatible boost version on Fedora 30

Compilation on Fedora 30 fails:
```
  --> Entering sub-directory shell ---
      --- Compiling shell.cpp (C++)
/home/thofmann/code/llsf-refbox/src/shell/shell.cpp: In member function ‘void llsfrb_shell::LLSFRefBoxShell::client_msg(uint16_t, uint16_t, std::shared_ptr<google::protobuf::Message>)’:
/home/thofmann/code/llsf-refbox/src/shell/shell.cpp:820:76: error: no matching function for call to ‘boost::posix_time::seconds::seconds(float)’
  820 |       attmsg_endtime_ = now + boost::posix_time::seconds(am->time_to_show());
      |                                                                            ^
In file included from /usr/include/boost/date_time/posix_time/posix_time_types.hpp:16,
                 from /usr/include/boost/asio/time_traits.hpp:23,
                 from /usr/include/boost/asio/basic_deadline_timer.hpp:28,
                 from /usr/include/boost/asio.hpp:24,
                 from /home/thofmann/code/llsf-refbox/src/shell/shell.h:42,
                 from /home/thofmann/code/llsf-refbox/src/shell/shell.cpp:39:
/usr/include/boost/date_time/posix_time/posix_time_duration.hpp:57:16: note: candidate: ‘template<class T> boost::posix_time::seconds::seconds(const T&, typename boost::enable_if<boost::is_integral<T>, void>::type*)’
   57 |       explicit seconds(T const& s,
      |                ^~~~~~~
/usr/include/boost/date_time/posix_time/posix_time_duration.hpp:57:16: note:   template argument deduction/substitution failed:
/usr/include/boost/date_time/posix_time/posix_time_duration.hpp: In substitution of ‘template<class T> boost::posix_time::seconds::seconds(const T&, typename boost::enable_if<boost::is_integral<T>, void>::type*) [with T = float]’:
/home/thofmann/code/llsf-refbox/src/shell/shell.cpp:820:76:   required from here
/usr/include/boost/date_time/posix_time/posix_time_duration.hpp:57:16: error: no type named ‘type’ in ‘struct boost::enable_if<boost::is_integral<float>, void>’
/usr/include/boost/date_time/posix_time/posix_time_duration.hpp:53:30: note: candidate: ‘boost::posix_time::seconds::seconds(const boost::posix_time::seconds&)’
   53 |   class BOOST_SYMBOL_VISIBLE seconds : public time_duration
      |                              ^~~~~~~
```

--- a/src/shell/shell.cpp
+++ b/src/shell/shell.cpp
@@ -817,7 +817,9 @@ LLSFRefBoxShell::client_msg(uint16_t comp_id, uint16_t msg_type,
     }
     if (attmsg_has_endtime_) {
       boost::posix_time::ptime now(boost::posix_time::microsec_clock::local_time());
-      attmsg_endtime_ = now + boost::posix_time::seconds(am->time_to_show());
+      attmsg_endtime_ = now +
+        boost::posix_time::milliseconds(static_cast<long>(1000 *
+              am->time_to_show()));
     }
     if (attmsg_string_ != "") {
       log(llsf_log_msgs::LogMessage::LL_ERROR, "A", "%s", attmsg_string_.c_str());

